File: solutions/c#/LinkedIn/Education.cs
namespace LinkedInNamespace
{
    public class Education
    {
        public string School { get; set; }
        public string Degree { get; set; }
        public string FieldOfStudy { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }

        public Education(string school, string degree, string fieldOfStudy, string startDate, string endDate)
        {
            School = school;
            Degree = degree;
            FieldOfStudy = fieldOfStudy;
            StartDate = startDate;
            EndDate = endDate;
        }
    }
}

File: solutions/python/linkedin/education.py
class Education:
    def __init__(self, school: str, degree: str, start_year: int, end_year: int):
        self.school = school
        self.degree = degree
        self.start_year = start_year
        self.end_year = end_year

    def __str__(self) -> str:
        return f"{self.degree}, {self.school} ({self.start_year} - {self.end_year})"
